content/03.methods.md

```diff
@@ -48,6 +48,14 @@ We trained this model using scikit-learn's [@raw:scikit-learn] implementation of
 For each principal component we calculated its cosine similarity with  all tokens in our word2vec model's vocabulary.
 We report the top 100 positive and negative scoring tokens in the form of  word clouds, where the size of each word corresponds to the magnitude of similarity and color represents positive (blue) or negative (orange) association.
 
-### Recommending Journals/ bioRxiv Audience Analysis
-1. This title will update as analysis is completed
-2. This section will describe how the above process is conducted
+### Journal Recommendation
+We trained two k-nearest neighbor classifiers to recommend journals based on article content.
+The first classifier recommends journals based on articles in close proximity to the query and the second classifier recommends journals based on close proximity to each journal's centroid.
+We used PubMed Central's (PMC) open access dataset to train both models.
+We preprocessed this dataset by reserving articles that were found in both PMC and bioRxiv for our test set and used the rest for training.
+Furhermore, we filtered out journals that had less than 100 articles within PMC and downsampled the remaining journals to only have 100 articles as this prevents overrepresentation of mega journals.
+We compared two distance metrics (Eucledian distance and Manhattan distance) for both classifiers.
+We report our results in the form of accuracy; however, we considered a prediction to be positive if the correct journal appeared within the ten closest neighbors of a query.
+The results of the first classifier were generated using 10-fold cross validation on the downsampled training set.
```

Comment:
I wonder if it would make sense to have what you have at the very top to explain the two classifiers, then to fully explain the article-based classifier, then the journal-based one. There are some subtle differences in here that might come out more effectively that way.